Running in wrapper mode for '192'

Running python job.

[name: "CPU-QUOTA"
int_val: 42
int_val: 48
int_val: 25
int_val: 97
int_val: 47
int_val: 99
int_val: 89
, name: "DISK"
int_val: 29
int_val: 5
int_val: 35
int_val: 8
int_val: 36
int_val: 40
int_val: 17
, name: "MEMORY"
int_val: 12
int_val: 15
int_val: 26
int_val: 8
int_val: 7
int_val: 24
int_val: 32
, name: "NET"
int_val: 35
int_val: 16
int_val: 39
int_val: 32
int_val: 21
int_val: 32
int_val: 35
]
The paramters are {u'CPU-QUOTA': array([42, 48, 25, 97, 47, 99, 89]), u'NET': array([35, 16, 39, 32, 21, 32, 35]), u'DISK': array([29,  5, 35,  8, 36, 40, 17]), u'MEMORY': array([12, 15, 26,  8,  7, 24, 32])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 120000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 81920.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 4194304.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 562500000.0
Recovering the memory utilization
Setting cpu quota to 105000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 179200.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 24326963.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 450000000.0
Recovering the memory utilization
Setting cpu quota to 62500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 199680.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 29360128.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 975000000.0
Recovering the memory utilization
Setting cpu quota to 105000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 179200.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 24326963.2
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 450000000.0
Recovering the memory utilization
Setting cpu quota to 62500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 199680.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 29360128.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 975000000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384749e0>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.38 seconds.
